schema/lastditch/image_test.go

```diff
@@ -0,0 +1,70 @@
+// Copyright 2015 The appc Authors
+//
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+//
+//     http://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+
+package lastditch
+
+import (
+	"reflect"
+	"testing"
+
+	"github.com/appc/spec/schema/types"
+)
+
+func TestImageManifestWithInvalidName(t *testing.T) {
+	invalidName := "example.com/test!"
+	imj := `
+		{
+		    "acKind": "ImageManifest",
+		    "acVersion": "0.6.1",
+		    "name": "` + invalidName + `"
```

Comment:
If you add some other valid fields in `imj` such as `"labels": [`, does the lastditch json parser successfully ignore them?